resources/language/resource.language.es_es/strings.po

```diff
@@ -126,11 +128,11 @@ msgstr "Biblioteca"
 
 msgctxt "#30026"
 msgid "Enable custom library folder"
-msgstr "Habilitar carpeta personalizada de la biblioteca"
+msgstr "Habilitar carpeta para biblioteca personalizada"
 
 msgctxt "#30027"
 msgid "Custom library path"
-msgstr "Ruta personalizada de la biblioteca"
+msgstr "Ruta"
```

Comment:
DON'T AGREE. Please respect original author's text.
Use instead: "Ruta **a la biblioteca personalizada**"

Reply:
DON´T AGREE??? I @boblo1 I see my cleanest and clearest solution.

![Sin título (2)](https://user-images.githubusercontent.com/24685253/79611289-9056ac00-80fa-11ea-8262-650da8f74eb1.png)
![Sin título](https://user-images.githubusercontent.com/24685253/79611272-89c83480-80fa-11ea-99d3-22c3e554fc31.png)